pkg/apis/work/v1alpha1/binding_types.go

```diff
@@ -30,6 +31,9 @@ type ResourceBindingSpec struct {
 	// Clusters represents target member clusters where the resource to be deployed.
 	// +optional
 	Clusters []TargetCluster `json:"clusters,omitempty"`
+	// ReplicasBinding represents the number of replicas in targetClusters have been set
+	// +optional
+	ReplicasBinding int64 `json:"replicasBinding,omitempty"`
```

Comment:
I don't understand the meaning of this field. Can you give more detailed comments?

Reply:
It is a record of the replicas we have set during  the scheduler.  If users update the replicas of workload the scheduler can find this change easily by comparing the ReplicasBinding and  Resource.Replicas。

Reply:
Or we can put it in the annotations of binding? In fact ,the record of ResourcePerReplicas is put in the  annotations

Reply:
> If users update the replicas of workload, the scheduler can find this change easily by comparing the ReplicasBinding and Resource.Replicas

Do you mean re-shcedule happens only when replica changes?

Not sure if we really need this fields, but if yes:
I suppose the field should be named with `ObservedReplicas` and should be in `.status.observedReplicas`. like:
```golang
	// ObservedReplicas represents the number of replicas observed by the scheduler.
	// +optional
	ObservedReplicas int64 `json:"observedReplicas,omitempty"`
```

Reply:
Reschedule will happen in the following  conditions.  First, the policy changes, and it is handled in ReconcileSchedule . Second, the replicas changes , so we need to record the replicas that scheduler has assigned . Third, the resource each pods use change, so we also  needs to record the ResourcePerReplicas.
```
        // ObservedReplicas represents the number of replicas observed by the scheduler.
	// +optional
	ObservedReplicas int64 `json:"observedReplicas,omitempty"`
        // ObservedResourcePerReplicas represents the resource each replicas needs observed by the scheduler.
       // +optional
       ObservedResourcePerReplicas corev1.ResourceList `json:"observedresourcePerReplicas,omitempty"`
```

Reply:
Let's wait for @kevin-wangzefeng's comments.

`int32` would be enough by the way.

Reply:
```yaml
spec:
  clusters:
    - name: cluster1
      replicas: 4
    - name: cluster3
      replicas: 6
  replicasBinding: 10
```
The `replicasBinding` always equal to the total replicas in each cluster, right?
`replicasBinding = spec.clusters[0].replicas + spec.cluster[1].replicas`
If it is the case, then the fields should be removed.